bundles/org.openhab.ui/web/src/assets/definitions/widgets/chart/index.js

```diff
@@ -286,7 +286,7 @@ export default {
     props: {
       parameterGroups: [actionGroup()],
       parameters: [
-        seriesTypeParameter('gauge', 'pie'),
+        seriesTypeParameter('bar', 'gauge', 'heatmap', 'line', 'pie', 'scatter'),
```

Comment:
As the order is shown in the docs and the UI: I would rather order it similar to below, i.e.:

line, bar, heatmap, scatter, gauge, pie

Reply:
I changed the order to alphabetical exactly because I realized that the order would be visible. I can change it to whatever order you'd like, but I'm not sure I "get" what order you're suggesting.

Reply:
If the order is that of the order in which they are documented at ECharts, the order should be:

line, bar, pie, scatter, heatmap, gauge

I have no idea how they came up with that order though - maybe it's simply the order in which the components were created/added?

Reply:
I was referring to the order in line 302 and 317 of this file.

> maybe it's simply the order in which the components were created/added?

I guess yes, but I would like to keep everything consistent with the status quo.